### PR TITLE
#110: Added conditional /customer in AtlasMockAPI

### PR DESCRIPTION
**Pull request by bonkey**
- Added conditional response from /customer based on “Authorized:” header: accepts any token
- Added AtlasMockAPIRunner – dumb-simple macOS command-line app to test AtlasMockAPI manually
  - Added swifter macOS library compilation in deps.rb
  - Added AtlasMockAPI_macOS target

### CODE REVIEW

**Comment by bonkey** [comment]
👍

File: AtlasMockAPI/AtlasMockAPI/HttpServer.swift [hunk 1]
```diff
@@ -12,6 +12,8 @@ enum HttpServerError: ErrorType {
 
 extension HttpServer {
 
+    var serverBundle: NSBundle { return NSBundle(forClass: AtlasMockAPI.self) }
```
Comment (hani-ibrahim) [inline AtlasMockAPI/AtlasMockAPI/HttpServer.swift]: Why returning a new instance each time it is used?
`let serverBundle = NSBundle(forClass: AtlasMockAPI.self)`?

Comment (bonkey) [inline AtlasMockAPI/AtlasMockAPI/HttpServer.swift]: that's extension... can't have properties.

Comment (hani-ibrahim) [inline AtlasMockAPI/AtlasMockAPI/HttpServer.swift]: Ah yeah true :+1: 
Didn't notice that 😃

**Pull request merged**